.gitmodules

```diff
@@ -11,3 +8,7 @@
 [submodule "src/tools/gdspy"]
 	path = src/tools/gdspy
 	url = https://github.com/heitzmann/gdspy.git
+[submodule "src/tools/pyyaml"]
+	path = src/tools/pyyaml
+	url = https://github.com/yaml/pyyaml.git
+	branch = release/6.1
```

Comment:
Question: if we are now able to use the release versions of pyyaml, does it still need to be a submodule, or can we just have it be a system requirement in the documentation/part of a dependency install script? This would get out of date pretty fast.

Reply:
That's also fine with me if you'd prefer

Reply:
yeah let's do that and only submodule dependencies that are not released or need specific versions.